licencjat.tex

```diff
@@ -793,77 +794,102 @@ \section{Metody porządkowania liniowego}
+\item $\licznosc{\{x \in X, \quad \licznosc{\{y \in X, \quad ySx \}}=1\}}=2 $
 \end{enumerate}
 gdzie:\\
-$S$ oznacza sąsiada względem relacji $\leq$, z kolei relacja $\leq$ jest porządkiem liniowym zbioru danych $X$.
-\end{wlasnosci}
-
-\begin{theorem}{Porządek liniowy zbioru danych $X$ jest porządkiem liniowym w sensie definicji 2.5.3.\\} 
-Porządek liniowy zbioru danych $X$ jest porządkiem liniowym w sensie definicji 2.5.3.
+$S$ oznacza sąsiada względem relacji $\leq$.
 \end{theorem}
+
 \begin{proof}
-By udowodnić to twierdzenie należy odwołać się do definicji częściowego porządku, a następnie skorzystać z definicji relacji liniowego porządku. Zatem, należy wykazać zwrotność, przechodniość, słabą antysymetryczność oraz spójność relacji. Niech dany będzie skończony, niepusty zbiór $X$ (przez zbiór $X$ będziemy rozumieli zbiór danych), oraz relacja $\leq$.\\
-Załóżmy, że zbiór $X \subset \mathbb{R}^2$ oraz $X=\{a_1, a_2,..., a_n\},\quad n\in \mathbb{N}$ oraz $a_i=(x_i,y_i) \in X,  i\in \mathbb{N}$\\
-Niech dane będą dwa obiekty $a_i, a_j \in X, \quad a_i=(x_i, y_i)  \wedge a_j=(x_j, y_j),  i, j\in \mathbb{N}$\\
-\begin{itemize}%[(alph)]
-\item (zwrotność)\\
-Zauważmy, że $\forall{a_i \in X}_{i \in \mathbb{N}} \quad a_i \leq a_i$, wtedy
-\begin{center}
- $(x_i, y_i) \leq (x_i, y_i)$, czyli $x_i \leq x_i \wedge y_i \leq y_i$. \\
-\end{center}
-Zatem relacja $\leq$ jest zwrotna.
-\item (słaba antysymetryczność)\\
-Niech dane będą obiekty $a_i, a_j \in X$ wówczas 
-\begin{center}
-$a_i \leq a_j \quad \wedge \quad a_j \leq a_i$ czyli\\
-$(x_i, y_i) \leq (x_j, y_j) \quad \wedge \quad  (x_j, y_j) \leq (x_i, y_i)$ zatem \\
-$(x_i \leq x_j \wedge y_i \leq y_j) \wedge (x_j\leq x_i \wedge y_j \leq y_i)\quad  \Rightarrow \quad x_i=x_j \wedge y_j=y_i \quad \Rightarrow \quad (x_i,y_i)=(x_j,y_j)$
-\end{center}
-Zatem relacja $\leq$ jest słabo antysymetryczna.
-\item (przechodniość)\\
-Niech dane będą obiekty $a_i, a_j, a_k \in X$, takie że $\quad a_i \leq a_j \quad  \wedge \quad a_j \leq a_k$
-\begin{center}
-$(x_i, y_i) \leq (x_j, y_j) \quad \wedge \quad  (x_j, y_j) \leq (x_k, y_k)$ czyli\\ 
-$(x_i \leq x_j \wedge y_i \leq y_j) \wedge (x_j \leq x_k \wedge y_j \leq y_k) \quad \Rightarrow$\\
-$x_i \leq x_k \wedge y_i \leq y_k \quad \Rightarrow (x_i,y_i) \leq (x_k,y_k)$\\
-\end{center}
-Zatem $\forall_{a_i, a_j, a_k \in X}\quad a_i \leq a_k,  $ stąd relacja $\leq$ jest przechodnia. 
-\end{itemize}
-Na podstawie powyższych punktów, widać iż para $(X,\leq)$ jest zbiorem częściowo uporządkowanym, sprawdzimy teraz czy relacja ta jest spójna.
+Udowodnimy kolejne własności.
+\begin{enumerate}
+\item Niech $x \in X$. Przypuśćmy na początek, że $\licznosc{\{y \in X, ySx\}} = 0$, tzn. że obiekt $x$ nie posiada sąsiadów w tej relacji. Nasz zbiór $X$ jest jednak conajmniej dwuelementowy zatem istnieje element $y \in X$ i $ x \neq y$. Wobec spójności linowego porządku z Definicji \ref{def-porzadek-liniowy} zachodzi wtedy
+$$
+x \leq y \lor y \leq x.
```

Comment:
operatory logiczne takie nie , lub , i piszemy używając 
nie -> not
lub -> or
i -> and
W LaTeX ich komendy zaczynają się od l jak logika czyli mamy \land do operatora i